src/metabase/models/action.clj

```diff
@@ -73,6 +87,7 @@
 (defn insert!
   "Inserts an Action and related HTTPAction or QueryAction. Returns the action id."
   [action-data]
+  (check-data-apps-enabled)
```

Comment:
Wouldn't pre-insert throw anyway?

Reply:
Ya, I figured it was a bit more clear to have it here, since the Action models are a bit unidiomatic.